src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumesInMemoryRepository.cs

```diff
@@ -113,5 +113,10 @@ public void Update(int id, Resume updateObject)
         {
             throw new NotImplementedException();
         }
+
+        public IEnumerable<Resume> GetByName(string name)
+        {
+            return Resumes;
```

Comment:
this method should be returning only the resumes that match the name, remember is an actual repo, is not a mock

Reply:
Done